docs/zh-CN/README.md

```diff
@@ -327,92 +323,90 @@ ThirdParty
 
 ![优雷卡屏幕截图](../../screenshots/promo_eureka.png)
 
-### [radar](../../ui/radar) 模块
+### [Radar](../../ui/radar) 悬浮窗
 
-要使用该模块，定位到 cactbot 下面的 **ui/radar/radar.html** 或使用 `Cactbot Radar` 预设。
+要使用该悬浮窗，定位到 Cactbot 下面的 **ui/radar/radar.html** 或使用 `Cactbot Radar` 预设。
 
-该模块可让您发现附近的狩猎怪（S 级，A 级等）。当一个狩猎怪出现时，该模块会显示一个箭头（基于角色面向）指向该狩猎怪并会显示您与该狩猎怪之间的距离。
+该悬浮窗可让你发现附近的狩猎怪（S 级，A 级等）。当一个狩猎怪出现时，该模块会显示一个箭头（基于角色面向）指向该狩猎怪并会显示你与该狩猎怪之间的距离。
 
-控制面板中的选项可以设置显示哪位玩家开了狩猎怪，还可以配置 radar 模块的显示方式。您还可以为不同等级的狩猎怪设置自定义选项（例如为 S 级怪发出声音，但对 B 级怪保持沉默），或为您想监控的任何目标名称添加自定义触发器。
+控制面板中的选项可以设置显示哪位玩家开了狩猎怪，还可以配置 Radar 模块的显示方式。你还可以为不同等级的狩猎怪设置自定义选项（例如用声音提示 S 级怪，但是忽略 B 级怪），或为你想监控的任何目标名称添加自定义触发器。
 
-有关更多选项，请参见`cactbot/user/radar-example.js`文件。
+有关更多选项，请参见 `cactbot/user/radar-example.js` 文件。
 
 ![雷达屏幕截图](../../screenshots/promo_radar.png)
 
-### [dps](../../ui/dps) 统计模块
+### [DPS](../../ui/dps) 统计悬浮窗
 
-cactbot 可以与为 OverlayPlugin 的数据统计功能设计的任何 DPS 统计悬浮窗共同使用，并可以选择通过 cactbot 附加的 Javascript API 扩展更多功能。cactbot 还可以在团灭时自动停止统计，因此您可以将 ACT 的战斗时间配置为无限。
+Cactbot 可以与为 OverlayPlugin 的数据统计功能设计的任何 DPS 统计悬浮窗共同使用，并可以选择通过 Cactbot 附加的 JavaScript API 扩展更多功能。Cactbot 还可以在团灭时自动停止统计，因此你可以将 ACT 的战斗时间配置为无限。
 
-[xephero](../../ui/dps/xephero)DPS 统计悬浮窗具有在多列显示对副本的每个阶段的 DPS 数据进行分段的功能。在下面的截图中，各阶段分别命名为 B1、B2、B3。它们会在 4 人本的 BOSS 自动生成，也可以用来区分 Raid 副本的阶段。
-要使用该悬浮窗，定位到 cactbot 下面的 **ui/dps/xephero/xephero-cactbot.html**
+[xephero](../../ui/dps/xephero) DPS 统计悬浮窗采用与 ACT 原生一致的统计模型。
 
 ![xephero 屏幕截图](../../screenshots/xephero.png)
 
-[rdmty](../../ui/dps/rdmty)DPS 统计悬浮窗为 4.X 职业进行了适配更新，并为了匹配[fflogs](http://fflogs.com)进行了重新着色。
-要使用该悬浮窗，定位到 cactbot 下面的 **ui/dps/rdmty/dps.html**
+[rdmty](../../ui/dps/rdmty) DPS 统计悬浮窗采用与 ACT 原生一致的统计模型，并采用了 [fflogs](http://fflogs.com) 的颜色策略。
 
 ![rdmty 屏幕截图](../../screenshots/rdmty.png)
 
-### [pull counter](../../ui/pullcounter) 模块
+### [Pull Counter](../../ui/pullcounter) 悬浮窗
 
-这个小模块可以显示您在高难度副本中当前的重试次数。 此功能是为主播或查看录屏的玩家准备的。 通过这个数字，您可以轻松浏览视频并查找到特定的某场战斗来进行检阅。
+这个小悬浮窗可以显示你在高难度副本中当前的重试次数。此功能是为主播或经常录屏的玩家准备的。通过这个数字，你可以轻松浏览视频并查找到特定的某场战斗来进行检阅。
 
-您可以通过在游戏聊天窗口中键入 `/echo pullcounter reset` 来重置当前副本 / 区域的重试计数。您也可以直接在 `%APPDATA%\Advanced Combat Tracker\Config\RainbowMage.OverlayPlugin.config.json` 文件中修改该计数。
+你可以通过在游戏聊天窗口中键入 `/echo pullcounter reset` 来重置当前副本 / 区域的重试计数。你也可以直接在 `%APPDATA%\Advanced Combat Tracker\Config\RainbowMage.OverlayPlugin.config.json` 文件中修改该计数。
 
 ![开怪计时截图](../../screenshots/pullcounter.png)
 
-### [test](../../ui/test) 模块
+### [Test](../../ui/test) 悬浮窗
 
-要使用该模块，定位到 cactbot 下面的 **ui/test/test.html** 或使用 `Cactbot Test` 预设。
+要使用该悬浮窗，定位到 Cactbot 下面的 **ui/test/test.html** 或使用 `Cactbot Test` 预设。
 
-该模块仅用于 cactbot 所用变量的可视化测试，并不适合在正常游戏时使用。该模块可用来测试所有数据的获取是否正常、模块能否按照预期正常工作，或用来调试悬浮窗的错误。
+该悬浮窗仅用于 Cactbot 所用变量的可视化测试，并不适合在正常游戏时使用。该模块可用来测试所有数据的获取是否正常、模块能否按照预期正常工作，或用来调试悬浮窗的错误。
 
 ![test 屏幕截图](../../screenshots/test.png)
 
 ## 疑难解答
 
-您可以在[此链接](./FAQ-Troubleshooting.md)中查看 Cactbot 的常见问题解答。
+你可以在 [此链接](./FAQ-Troubleshooting.md) 中查看 Cactbot 的常见问题解答。
```

Comment:
The link to `./FAQ-Troubleshooting.md` here is incorrect (because there is no `./docs/zh_CN/FAQ-Troubleshooting.md` file).

Reply:
Good catch, I change it to link to the English version right now.